x-pack/plugins/session_view/public/types.ts

```diff
@@ -20,6 +20,19 @@ export interface SessionViewDeps {
   // if provided, the session view will jump to and select the provided event if it belongs to the session leader
   // session view will fetch a page worth of events starting from jumpToEvent as well as a page backwards.
   jumpToEvent?: ProcessEvent;
+  // Callback to open the alerts flyout
+  loadAlertDetails?: (
+    alertUuid: string,
+    // Callback used when alert flyout panel is closed
+    handleOnAlertDetailsClosed: () => void
+  ) => void;
+}
+
+export interface UpdateAlertStatus {
```

Comment:
this type seems too generic and doesn't seem to tell much about it, can it be improved?